src/iconsManifest/supportedExtensions.ts

```diff
@@ -4568,6 +4568,12 @@ export const extensions: IFileCollection = {
       filename: true,
       format: FileFormat.svg,
     },
+    {
+      icon: 'wgsl',
+      extensions: [],
+      languages: [languages.wgsl],
```

Comment:
Is there an extension for wgsl? Or is it native to vscode?
Just wondering if it's actually a language ID or if you can remove it!

Reply:
The [WGSL](https://marketplace.visualstudio.com/items?itemName=PolyMeilex.wgsl) extension defines the language id in its package.json

So I assumed it would be correct to add it as a language rather than just using the file extension, please correct me if this is wrong.


Snippet from the extensions package.json:
```json
"languages": [
      {
        "id": "wgsl",
        "aliases": [
          "WGSL",
          "wgsl"
        ],
        "extensions": [
          ".wgsl"
        ],
        "configuration": "./language-configuration.json"
      }
    ],
```

Reply:
That’s perfect :)